pkg/taskdir/definitions/def_0_2.go

```diff
@@ -0,0 +1,66 @@
+package definitions
+
+import (
+	"github.com/airplanedev/cli/pkg/api"
+)
+
+type Definition_0_2 struct {
+	Slug           string             `yaml:"slug"`
+	Name           string             `yaml:"name"`
+	Description    string             `yaml:"description,omitempty"`
+	Image          string             `yaml:"image,omitempty"`
+	Command        []string           `yaml:"command,omitempty"`
+	Arguments      []string           `yaml:"arguments,omitempty"`
+	Parameters     api.Parameters     `yaml:"parameters,omitempty"`
+	Constraints    api.RunConstraints `yaml:"constraints,omitempty"`
+	Env            api.TaskEnv        `yaml:"env,omitempty"`
+	ResourceLimits api.ResourceLimits `yaml:"resourceLimits,omitempty"`
+	Repo           string             `yaml:"repo,omitempty"`
+	Timeout        int                `yaml:"timeout,omitempty"`
+
+	Manual *ManualDefinition `yaml:"manual,omitempty"`
+	Deno   *DenoDefinition   `yaml:"deno,omitempty"`
+	Docker *DockerDefinition `yaml:"docker,omitempty"`
+	Go     *GoDefinition     `yaml:"go,omitempty"`
+	Node   *NodeDefinition   `yaml:"node,omitempty"`
+	Python *PythonDefinition `yaml:"python,omitempty"`
+
+	// Root is a directory path relative to the parent directory of this
+	// task definition which defines what directory should be included
+	// in the task's Docker image.
+	//
+	// If not set, defaults to "." (in other words, the parent directory of this task definition).
+	//
+	// This field is ignored when using the pre-built image builder (aka "manual").
+	Root string `yaml:"root,omitempty"`
+}
+
+type ManualDefinition struct {
+	Config map[string]string
```

Comment:
The config we would need for the manual "builder" mirrors what we have in the UI:

![image](https://user-images.githubusercontent.com/2907397/116443685-a90a9f00-a7ef-11eb-824e-ea4865b147f4.png)


```suggestion
	Image string `yaml:"image"`
	Command []string `yaml:"command,omitempty"`
```

Reply:
Also we should probably rename `Entrypoint` in the UI here to `Command` to mirror Docker terminology.

Reply:
For the manual builder, we don't actually set anything in `builder` or `builderConfig`. Manual is what all tasks looked like before we introduced builders -- you would bring your own pre-built Docker image that we would run. So the config instead lives in the `image`/`command` top-level fields, but I like how your PR will hide that distinction in the file format :)

Reply:
We would then need to pull out `image/command` from the manual config here: https://github.com/airplanedev/cli/pull/105/files#diff-ee891759dd8b9f7107c066f2b0545cd4e07fc73403cbc252f482a0c5c466b53e

Noting that `image` and `command` can/should be empty strings for the other builders (the former defaults to the Airplane Registry and the latter defaults at runtime to the command from the Docker image).